pytorch_lightning/core/lightning.py

```diff
@@ -1054,7 +1054,7 @@ def test_epoch_end(self, outputs):
                     self.log('final_metric', final_value)
         """
 
-    def predict(self, batch: Any, batch_idx: int, dataloader_idx: Optional[int] = None):
+    def predict_step(self, batch: Any, batch_idx: int, dataloader_idx: Optional[int] = None):
```

Comment:
this break backwards compatibility right?

Reply:
Yes, but it is intended. The predict API hasn't been added to the documentation yet as it is likely to changed. However, after this PR, it should be final and I will make a PR to add it to the documentation.